layout/theme.liquid

```diff
@@ -46,26 +46,19 @@
         --font-heading-weight: {{ settings.type_header_font.weight }};
 
         --color-base-text: {{ settings.colors_text }};
-        --color-base-text-rgb: {{ settings.colors_text | color_to_rgb | remove: 'rgb(' | remove: ')' }};
+        --color-base-text-rgb: {{ settings.colors_text.red }}, {{ settings.colors_text.green }}, {{ settings.colors_text.blue }};
```

Comment:
Is there opportunity to clean this up even further and not have two versions of the same colour? I assume we can leverage one custom property for e.g. `--color-base-text` and have that map to the RGB equivalent.

Reply:
Yes. I would preferably address that in a separate PR. That along with the 2nd point under ["other considerations"](https://github.com/Shopify/dawn/pull/83#issue-680977592). 

For this PR, the goal is to get rid of `remove: 'rgb(' | remove: ')'` and provide a quick fix for #52